azure-keyvault/src/main/java/com/microsoft/azure/keyvault/models/BackupSecretResult.java

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
This is a model from the updated spec we should remove it from the refactoring, and add back in a subsequent update.